_posts/en/2019-01-20-rbf-in-the-wild.md

```diff
@@ -0,0 +1,242 @@
+
+![GreenAddress wallet shows a bump fee dropdown with fee options screenshot](/img/posts/rbf-in-the-wild/rbf-green-multiple-bump-options.png)
+
+GreenAddress allows the user, who is trying to bump an existing transaction, to choose their fee bump level based on some preselected options. Fiat (USD) amounts are also shown which means users do not need to do the conversion manually. The GreenAddress bump fee dialog also has an estimation about the current fee confirmation level.
+
+### Bitcoin Core: Showing Original Tx Post Confirm
+
+![Bitcoin Core showing 'conflicted' transaction mouse over screenshot](/img/posts/rbf-in-the-wild/rbf-bitcoin-core-show-original.png)
+
+Bitcoin Core shows the original transaction as failed (with X icon) after a bumped transaction replaces it and confirms (with checkmark icon). A tooltip shows the address that the replacement transaction came in to, which in this and many cases is the same as the original.
+
+### GreenAddress: Visual RBF Flag in Tx List
+
+![GreenAddress labeling of RBF transaction screenshot](/img/posts/rbf-in-the-wild/rbf-green-rbf-label.png)
+
+GreenAddress shows a visual RBF flag (“replaceable”) in the Tx list. Also noted is the fact that there is a double spend since a replacement transaction has already been broadcast in this example.
+
+### Bitpay: Overt RBF Warning
+
+![Bitpay payment receipt showing RBF warnings screenshot](/img/posts/rbf-in-the-wild/rbf-bitpay-overt-warnings.png)
+
+Bitpay shows an explicit warning for RBF transactions saying that there may be a delay in processing. Bitpay also recommends "To avoid this delay in the future disable the RBF setting in your wallet". These messages are misleading and also infer that unconfirmed "non RBF" signaled transactions are not replaceable.
+
+Any unconfirmed transaction risks being replaced for a variety of reasons including broadcasting 2 transactions spending the same UTXO, directly mining a different transaction spending the same UTXO, or working with a miner to get a different transaction spending the same UTXO included in a block.
+
+### Explorer blocktrail.com: RBF Visual Label
+
+![blocktrail.com explorer transaction details showing optin rbf label screenshot](/img/posts/rbf-in-the-wild/rbf-blocktrain-rbf-label.png)
+
+Blocktrail shows an orange Opt-in RBF label and also a mouseover detail for the user providing more information.
+
+### Samourai: Silently Fail Transaction Bump
+
+![Samourai wallet increase transaction fee dialog screenshot](/img/posts/rbf-in-the-wild/rbf-samourai-fail-bump.png)
+
+Similar to the Bitcoin Core example above, there was a problem with fee bumping the transaction. However, in this case, there was a silent failure and no message for the user on what happened. There were sufficient funds in the wallet for a replacement transaction.
+
+Additional testing showed that fee bumping is possible in Samourai and perhaps this is an edge case in the code involving the change address or additional inputs.
```

Comment:
Similar to the Bitcoin Core, I think we should note whether or not this bug was reported to the wallet's devs.

Edit: there seem to be a few more cases where this same basic concern about ensuring that we reported to upstream applies.